neo4j/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # CHANGELOG - neo4j
 
-## 2.0.2 / 2023-04-28
```

Comment:
Small nit, you should just appent v3.0.0 to the CHANGELOG, not replace 2.0.2 with 3.0.0.

Reply:
there never was a 2.0.2 released.

That's why it is this way.

Reply:
We [released a 2.0.2](https://github.com/DataDog/integrations-extras/pull/1835) for neo4j when we had to remove a config value for a few integrations

Reply:
My bad. Put that information into the CHANGELOG.

Thanks for pointing it out.